rustls/src/client/client_conn.rs

```diff
@@ -401,7 +404,13 @@ impl Default for Resumption {
     /// Create an in-memory session store resumption with up to 256 server names, allowing
     /// a TLS 1.2 session to resume with a session id or RFC 5077 ticket.
```

Comment:
I think this rustdoc comment should be updated to mention that for no-std it actually disables resumption.

Reply:
(Though maybe of limited value since this comes back in phase ii)

Reply:
I did it anyway :P

Reply:
Thanks :-)